### PR TITLE
Void

### PR DESCRIPTION
**Pull request by abebeos**
### Description

Build/Test should be in the readme, for a real quickstart

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,18 @@ Aptos is a layer 1 blockchain bringing a paradigm shift to Web3 through better t
...
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+cargo build
+cargo test
+```
+
+### Resources
+
 * [Aptos Labs](https://aptoslabs.com/)
```
Comment (clay-aptos) [inline README.md:28]: @davidiw should this link to aptoslabs.com come out of our README?

**Review by clay-aptos — APPROVED** [review]
Thanks for the additions!

File: README.md (hunk 1 above)
Comment (abebeos) [reply README.md:30]: I found the links ok, not too much.

Aptos Labs are the makers, so i would say a link  belongs in the readme.

Comment (clay-aptos) [reply README.md]: I removed it as we're working to make all of this clearer.

Comment (abebeos) [reply README.md]: > Make all of this clearer.

Aptos Labs are the makers, 

Aptos Foundation the copyright holders,

File: README.md [hunk 2]
```diff
@@ -13,13 +13,29 @@ Aptos is a layer 1 blockchain bringing a paradigm shift to Web3 through better t
...
+### Quick
+
+```sh
+git clone https://github.com/aptos-labs/aptos-core.git
+cd aptos-core
+./scripts/dev_setup.sh
+cargo build
+cargo test
+```
+
```
Comment (davidiw) [inline README.md:25]:
```suggestion
```

Comment (davidiw) [reply README.md:25]: I don't think we need this content duplicated here.

**Review by davidiw — APPROVED** [review]
rest looks good to me!

**Comment by abebeos** [comment]
> rest looks good to me!

Hey, what "rest"?

The quickstart was the main thing (a quasi-standard for a code repo)

File: README.md (hunk 2 above)
Comment (abebeos) [reply README.md]: (process: You should not "comment close", things disappear easily that way)

its a code repo. Quickstarts are quasi-standard, and slight duplication is never avoidable.

**Comment by abebeos** [comment]
Not my intended/authored PR anymore, closing.

**Pull request closed without merging**